Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -193,6 +194,9 @@ def Set(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
 def Test(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     ShowMof('TEST', ConfigType, ConfigID, Contents, Ensure, ContentChecksum)
     retval = 0
+    if not os.path.exists(AGENT_SCRIPT_PATH):
```

Comment:
Why do you add this check?
Since the script is included in the module and the module is installed, the script is definitely there.